### DISCUSSION TITLE
[FEAT] Add projects using Textual to documentation?

### DISCUSSION THREAD

**Discussion by TomJGooding** [Textualize/textual#1991] [Ideas]
The Textualize website has a gallery of [Projects using Textual](https://www.textualize.io/textual/gallery) and I know @davep has also started a [personal collection ](https://github.com/davep/transcendent-textual).

Are there plans to include some examples of projects using Textual to the documentation?  

The example apps in this repo are great for getting started (although I do think the code for these could be more discoverable - see #1755 & #1766), but for me it was checking out some 'real' projects that really gave me a better understanding not only of what Textual is capable of, but also the API in general from looking at some code in a bigger context.

Just a thought - even if there was a link to the textualize.io gallery somewhere in the docs.

**Comment by davep** [comment]
We'll be reworking our main website in the near future and will be retiring the gallery. Our preference is to not maintain a list of applications built with Textual in the docs.

Meanwhile though I aim to carry on maintaining and curating my list, which we may mention somewhere visible at some point.

**Reply by TomJGooding** [reply]
Fair enough, it was just an idea. (Will try to remember to post in Discussions rather than raise an Issue in future!)

**Reply by davep** [reply]
> Will try to remember to post in Discussions rather than raise an Issue in future!

Don't worry too much about that; pick which feels right, moving something is never judgement. ;-)
